<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IconPickerDialog</class>
 <widget class="QWidget" name="IconPickerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">.btn_icon {
	background: #F4F4F4;
	border: none;
}
btn_icon:hover {
	background:  lightblue;
}

.dialog_frame {
        background: #F4F4F4;
        border: none;
}
.frame_header_dialog {
                background: #F4F4F4;
       border-bottom: 1px solid gray;
}
.label_h1, logo {
        background: #F4F4F4;
}

QTextEdit {
        background: white;
        color: black;
}
Spacer {
        background: #F4F4F4;
}
QLabel {
    color: black;
       background: #F4F4F4;
}
QScrollArea {
	background: #F4F4F4;
	border: none;
}
QGroupBox {
    border: 1px solid gray;
    border-radius: 5px;
    padding-top: 10px;
       margin-top: 8px;
}
QGroupBox::title {
        color: black;
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0px 10px;
}
QComboBox, QAbstractItemView{
        color: black;
        background: white;
}
QComboBox::Items {
        color: black;
}
QLineEdit {
        color: black;
        background: white;
}

.pushButton_action {
        border: 1px solid rgb(170,0,0);
        border-radius: 8px;
        padding: 6px;
        color: rgb(170,0,0);
        font-weight: bold;
        font-size: 12px;
}
QListWidget::item:selected {
    background-color: lightblue;
        color: black;
}
</string>
  </property>
  <property name="class" stdset="0">
   <string>dialog_frame</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_header">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <property name="class" stdset="0">
      <string>dialog_frame</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="QLabel" name="label_header">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700;&quot;&gt;Icons&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_content">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <property name="class" stdset="0">
      <string>dialog_frame</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox_icons">
        <property name="title">
         <string>Auswahlmöglichkeiten</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QScrollArea" name="scrollArea">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>462</width>
              <height>254</height>
             </rect>
            </property>
            <property name="class" stdset="0">
             <string>dialog_frame</string>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_footer">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <property name="class" stdset="0">
      <string>dialog_frame</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_quit">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Abbrechen</string>
        </property>
        <property name="class" stdset="0">
         <string>pushButton_action</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_save">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Übernehmen</string>
        </property>
        <property name="class" stdset="0">
         <string>pushButton_action</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
